app/models/peeps.rb

```diff
@@ -0,0 +1,10 @@
+
+class Peep
+
+	include DataMapper::Resource
+
+	property :id, Serial
+	property :peep, String, :length => 1..140, required: true
```

Comment:
Style/HashSyntax: Use the new Ruby 1.9 hash syntax. (https://github.com/bbatsov/ruby-style-guide#hash-literals)